db/migrate/20230921204534_remove_status_from_user_mentee_application.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveStatusFromUserMenteeApplication < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :user_mentee_applications, :status, :integer
+  end
```

Comment:
Oh yes, I wasn't supposed to push this changes, i was testing something. This will be removed next push. 

I just submitted a different PR to remove this column that didn't include a migration

Reply:
just pull the latest master into your branch and it will fix this.